lib/Core/Executor.cpp

```diff
@@ -1914,14 +1915,17 @@ void Executor::executeInstruction(ExecutionState &state, KInstruction *ki) {
 
   case Instruction::Invoke:
   case Instruction::Call: {
+    // Ignore debug intrinsic calls
+    if (isa<DbgInfoIntrinsic>(i))
+      break;
     CallSite cs(i);
 
     unsigned numArgs = cs.arg_size();
     Value *fp = cs.getCalledValue();
     Function *f = getTargetFunction(fp, state);
 
     // Skip debug intrinsics, we can't evaluate their metadata arguments.
-    if (f && isDebugIntrinsic(f, kmodule.get()))
+    if (isa<DbgInfoIntrinsic>(i))
```

Comment:
This check is now obsolete

Reply:
@jbuening could you follow up with a separate PR?  Given that this is minor and we'd like to release today, I'd like to address it later on.

Reply:
@ccadar Yes, sure! :+1: